app/controllers/question.rb

```diff
@@ -1,17 +1,28 @@
 get '/questions' do
   @questions = Question.all
   erb :'/questions/index'
-end 
+end
 
 get '/questions/new' do
   erb:'/questions/new'
 end
 
 
-get '/questions/:id' do 
+get '/questions/:id' do
   @question = Question.find_by(id: params[:id])
   @answers = Answer.where(question_id: params[:id])
   erb :'/questions/show'
-end 
-
+end
 
+post '/questions' do
+  require_login
+  @question = Question.new(params[:question])
+  @question.author = current_user
+  # binding.pry
+  if @question.save
+    redirect "/questions/#{@question.id}"
+  else
+    @errors = @question.errors.full_messages
+    erb:'/questions/new'
+  end
```

Comment:
looks good you could remove the @ sign from questions because you don't need to carry it over to anything